.github/workflows/project.yml

```diff
@@ -41,4 +41,7 @@ jobs:
       - uses: ./.github/actions/node/setup
       - run: yarn install
       - run: yarn type:test
+      - uses: actions/setup-node@v3
```

Comment:
Why not just default to `14` in both release lines for `setup` instead? The version from `setup` is usually used for tooling, and it's likely tooling dropped support for EOL versions of Node anyway.

Reply:
This ended up getting reverted; it was part of me experimenting with the failures.